saleor/userprofile/i18n.py

```diff
@@ -95,15 +93,15 @@ class Meta:
         exclude = []
         labels = {
             'first_name': pgettext_lazy(
-                'First name', 'Given name'),
+                'Person\'s name', 'Given name'),
             'last_name': pgettext_lazy(
-                'Last name', 'Family name'),
+                'Person\'s surname', 'Family name'),
```

Comment:
I suggest both use "Personal name", there's no reason to provide a unique context for family name.